x-pack/legacy/plugins/ml/public/services/job_service.d.ts

```diff
@@ -31,6 +33,7 @@ declare interface JobService {
   ): Promise<any>;
   createResultsUrl(jobId: string[], start: number, end: number, location: string): string;
   getJobAndGroupIds(): ExistingJobsAndGroups;
+  searchPreview(job: CombinedJob): Promise<any>;
```

Comment:
question: is there any interface for elasticsearch response?

Reply:
yes, `SearchResponse`. i'll update this.